go.mod

```diff
@@ -37,10 +37,9 @@ require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/docker/distribution v2.8.1+incompatible // indirect
-	github.com/docker/docker v1.6.1 // indirect
+	github.com/docker/docker v1.4.2-0.20190924003213-a8608b5b67c7 // indirect
```

Comment:
Why not use v20.10.xx

Reply:
Because this version is imported by `github.com/containerd/containerd v1.6.6`

Reply:
Hi, @AkihiroSuda , I have upgraded `github.com/docker/docker` to `v20.10.17+incompatible` :)